csrc/scheduler/expr_eval_sched.cpp

```diff
@@ -0,0 +1,58 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+
+#include <ir/utils.h>
+#include <scheduler/debug_utils.h>
+#include <scheduler/expr_eval_sched.h>
+#include <scheduler/registry_utils.h>
+
+namespace nvfuser {
+
+template <typename... Args>
+void vlog(const Args&... args) {
+  scheduler_debug_utils::log("[Expression Evaluator Scheduler] ", args...);
+}
```

Comment:
I think you could use a variadic macro here. I dont think there's any functional difference so its up to you but I could be wrong.